docs/Functions.md

```diff
@@ -0,0 +1,58 @@
+## Functions
+*This file is automatically generated from the
+            [def files](/onnx/defs) via [this script](/onnx/defs/gen_doc.py).
+            Do not modify directly and instead edit function definitions.*
+## ai.onnx (default)
+  * <sub>experimental</sub><a href="#MeanVarianceNormalization">MeanVarianceNormalization</a>
+
+
+
+### <sub>experimental</sub> <a name="MeanVarianceNormalization"></a><a name="meanvariancenormalization">**MeanVarianceNormalization**</a>
+
+  A MeanVarianceNormalization Function: Perform mean variance normalization on the input tensor X
```

Comment:
Need detailed description and mathematical formulations. As a standard, a clear document is very important. The supported input shape is likely any dimension. Please describe at least [N, C, H, W] and N-D cases.